### PR TITLE
[PRD-5563] - Bold font in windows menu

### PR DESCRIPTION
**Pull request by RPAraujo**
Cleaned up SystemInformation.java
Fixed bug and added unit tests

### CODE REVIEW

**Comment by RPAraujo** [comment]
@pamval, @pedrofvteixeira Plase review
I opted for keeping two commits, instead of squashing them, for better readability of the changes made

**Comment by pedrofvteixeira** [comment]
One commit for checkstyle and license header cleanup, and another with the fix alone, correct ?
I'm good with these changes.  

@pamval, your call.

File: libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java [hunk 1]
```diff
@@ -32,13 +32,30 @@
...
+  }
 
   public SystemInformation() {
   }
 
   public void logSystemInformation() {
-    if ( logger.isInfoEnabled() ) {
-      logger.info( getSystemInformation() );
+    if ( getLogger().isInfoEnabled() ) {
+      getLogger().info( getOtherProperties() );
```
Comment (pamval) [inline libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java]: Shouldn't this call getSystemInformation as before?

File: libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java [hunk 2]
```diff
@@ -89,9 +106,12 @@ private String formatLongPathList( String value ) {
     return value;
   }
 
+  public Map<String, String> getOtherPropertiesInst() {
```
Comment (pamval) [inline libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java]: Is this method used anywhere?

Comment (RPAraujo) [inline libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java]: No! Sorry, its leftover trash from when we where developing unit tests. I'll remove it.

File: libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java (hunk 1 above)
Comment (RPAraujo) [inline libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java]: Yes it should! Sorry for the slip-up. I must have changed it inadvertently during the refactoring. I'll fix it.

**Comment by pamval** [comment]
@RPAraujo check style violations. PLease review

**Comment by pamval** [comment]
And also please squash the two commits with the fix into one.

**Comment by RPAraujo** [comment]
@pamval, @pedrofvteixeira please review
Squashed the commits and added Powermockito to unit test the static method, this allowed removing the global static variable.

**Comment by tmorgner** [comment]
@pamval  @RPAraujo 

Sorry, dont agree with that fix. You are hijacking a logging method to do initialization. That is wrong. 

If you want to run code for report-designer when it boots up, then run it in ReportDesignerBoot or in the report-designer Main class. Apparently something in UIManager.getLookAndFeel is triggering the changed behaviour - so please find out which code exactly it is and then move it to the "ReportDesigner" class.

There is a "SetLookAndFeelTask" that is a much better place to hold this L&F initialisation.

**Comment by pamval** [comment]
@RPAraujo can you please look at this again, following Thomas comments?

**Pull request merged**